### PR TITLE
MM-10183: Adds docs for update channel scheme endpoint.

### PR DESCRIPTION
**Pull request by mkraft**
Docs for new API endpoint to update a channel's scheme.

### CODE REVIEW

File: v4/source/channels.yaml [hunk 1]
```diff
@@ -1330,3 +1330,56 @@
...
+      description: |
+        Set a channel's scheme.
+        
+        ##### Permissions
+        Must have `manage_system` permission.
+
+        __Minimum server version__: 4.10
+      parameters:
+        - name: channel_id
+          in: query
```
Comment (jespino) [inline v4/source/channels.yaml]: This must be path.

File: v4/source/channels.yaml [hunk 2]
```diff
@@ -1330,3 +1330,56 @@
...
+          description: Channel GUID
+          required: true
+          type: string
+        - name: scheme_id
+          in: query
```
Comment (jespino) [inline v4/source/channels.yaml]: this must be path.

File: v4/source/channels.yaml [hunk 3]
```diff
@@ -1330,3 +1330,56 @@
...
             // GetChannelUnread
             channelUnread, resp := Client.GetChannelUnread(<CHANNELID>, <USERID>)
+
+'/channels/{channel_id}/schemes/{scheme_id}':
+    put:
+      tags:
+        - channels
+      summary: Set a channel's scheme
+      description: |
+        Set a channel's scheme.
```
Comment (jespino) [inline v4/source/channels.yaml]: I would like to have a more clear explanation here. Without context is difficult to understand that you are setting the schema_id as schema of the channel.

File: v4/source/channels.yaml [hunk 4]
```diff
@@ -1330,3 +1330,64 @@
...
             // GetChannelUnread
             channelUnread, resp := Client.GetChannelUnread(<CHANNELID>, <USERID>)
+
+'/channels/{channel_id}/scheme':
+    put:
+      tags:
+        - channels
+      summary: Set a channel's scheme
+      description: |
+        Set a channel's scheme, to wit sets the scheme_id value of a channel record.
```
Comment (grundleborg) [inline v4/source/channels.yaml]: very Shakespearean ;)

Comment (jwilander) [reply v4/source/channels.yaml]: I'd prefer if we used more common language here. We have many second/third language english readers using these docs and I'm going to be honest I've never seen this phrase before

Comment (mkraft) [reply v4/source/channels.yaml]: I'm going to leave it because it's a perfectly good way to say "more specifically".

Comment (jwilander) [reply v4/source/channels.yaml]: Is that the goal of these docs, though? To me the goal is to be easy to read and understand so I think we should use more common language wherever possible to achieve that goal.

Comment (grundleborg) [reply v4/source/channels.yaml]: I think Joram has a point here. It's quite an obscure phrase (I've never encountered it outside of the theatre/English Literature classes at school), so I think it probably doesn't belong in a technical document that's going to be read by many folks for whom English is a second/third/etc language.

Comment (mkraft) [reply v4/source/channels.yaml]: I don't agree, sorry.

Comment (esethna) [reply v4/source/channels.yaml]: Sorry @mkraft, I agree with JW and GG here. Can we use a more common phrase? Our docs are not localized so we need to be as straight forward as possible.

**Comment by mkraft** [comment]
@esethna @grundleborg @jwilander Changed wording.

**Comment by jwilander** [comment]
@mkraft mind rebasing this and then it should be good to merge

**Comment by cpanato** [comment]
spoke in the pre-release channel

**Pull request merged**